test/runtest.py

```diff
@@ -260,6 +260,47 @@ class ClassA(object):
             ray.put(obj)
 
 
+def test_serialize_dataclasses(ray_start):
```

Comment:
Can we lump this in with the other serialization tests? Ideally we'd have a single test for serialization and not one `ray.init` call for every object that we want to test.